### DISCUSSION TITLE
Getting error 403 when trying to clone MaxPhisher repo??

### DISCUSSION THREAD

**Discussion by madsz1** [community/community#127417] [New to GitHub] [answered]
I'm trying to set up MaxPhisher in Ubuntu but I'm getting this error: 
Cloning into 'MaxPhisher'...
Username for 'https://github.com': madsz1
Password for 'https://madsz1@github.com':
remote: Access to this repository has been disabled by GitHub staff.
remote: If you are the repository owner, you can contact support via https://support.github.com/contact for more information.
fatal: unable to access 'https://github.com/KasRoudra/MaxPhisher/': The requested URL returned error: 403

When I try to directly run this: wget https://raw.githubusercontent.com/KasRoudra/MaxPhisher/main/maxphisher.py && python3 maxphisher.py
I get... :
--2024-06-05 14:57:26--  https://raw.githubusercontent.com/KasRoudra/MaxPhisher/main/maxphisher.py
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-06-05 14:57:27 ERROR 404: Not Found.

Could someone guide me in the write direction possibly I'm missing something. I'm new and inexperienced so it could be something painfully obvious to some. Any help is much appreciated.

**Comment by pedrolemos4** [comment]
Try to do `git config --global user.name "Your_Github_Account_Name"` and `git config --global user.email "Your_Github_Email"`
After this do git clone.
If it does not work with HTTP, generate a pair of keys with `ssh-keygen`, fill the information, put the id_rsa.pub that is located at `/home/user_name/.ssh` on your GitHub account SSH keys and try to clone it using SSH with the SSH option when trying to clone it.

**Reply by pedrolemos4** [reply]
Check my 2nd answer 1st

**Reply by TJd4vhVUHBCfHDU3c1hezYgtvmj4UiqGiD** [reply]
这个可以恢复钱包吗

**Comment by pedrolemos4** [comment, marked as answer]
Before everything, what is the repo you are trying to access? The repo on your command does not exist. You must go on the repo front page and select clone, copy the URL and do `git clone URL`

**Reply by madsz1** [reply]
Thank you for your reply, you were write with your latest post, the repo was wrong. It wasn't updated in the instructions I was following to match the URL which I proceeded to copy and it worked then thanks. Appreciate your reply.

**Reply by pedrolemos4** [reply]
No worries :) Can you please mark my answer as correct?

**Reply by Ayoade-i4v** [reply]
> Before everything, what is the repo you are trying to access? The repo on your command does not exist. You must go on the repo front page and select clone, copy the URL and do `git clone URL`

damnnn someone should make you president great problem solving i should have you as a friend so i can my bugs to fix lol what's your cell
 here's my mail budddy chukwuemekavictorudeze@gmail.com

**Comment by LKfollowings** [comment]
> أحاول إعداد MaxPhisher في Ubuntu ولكني أتلقى هذا الخطأ: الاستنساخ في 'MaxPhisher'... اسم المستخدم لـ ' https://github.com ': كلمة المرور madsz1 لـ ' [https://madsz1@github. com](https://madsz1@github.com) ': عن بعد: تم تعطيل الوصول إلى هذا المستودع من قبل موظفي GitHub. عن بعد: إذا كنت مالك المستودع، فيمكنك الاتصال بالدعم عبر https://support.github.com/contact لمزيد من المعلومات. فادح: غير قادر على الوصول إلى ' https://github.com/KasRoudra/MaxPhisher/ ': تم إرجاع خطأ URL المطلوب: 403
> 
> عندما أحاول تشغيل هذا مباشرة: wget https://raw.githubusercontent.com/KasRoudra/MaxPhisher/main/maxphisher.py && python3 maxphisher.py أحصل على... : --2024-06-05 14:57: 26-- https://raw.githubusercontent.com/KasRoudra/MaxPhisher/main/maxphisher.py حل Raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ... جارٍ الاتصال بـ Raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... متصل. تم إرسال طلب HTTP، في انتظار الرد... 404 غير موجود 2024-06-05 14:57:27 خطأ 404: غير موجود.
> 
> هل يمكن لأي شخص أن يرشدني في اتجاه الكتابة ربما أفتقد شيئًا ما. أنا جديد وعديم الخبرة لذا قد يكون الأمر واضحًا بشكل مؤلم للبعض. أي مساعدة هي محل تقدير كبير.

**Reply by Panin664356** [reply]
Bloquear todos los datos personales y clonados de github en todos los sitios web.

**Comment by TJd4vhVUHBCfHDU3c1hezYgtvmj4UiqGiD** [comment]
谁能够帮我恢复冷钱包，usdt 我忘记备份私钥，能解决的联系我qq438006422 成功有尝

**Comment by abobsasasasa18888** [comment]
> I'm trying to set up MaxPhisher in Ubuntu but I'm getting this error: 
> Cloning into 'MaxPhisher'...
> Username for 'https://github.com': madsz1
> Password for 'https://madsz1@github.com':
> remote: Access to this repository has been disabled by GitHub staff.
> remote: If you are the repository owner, you can contact support via https://support.github.com/contact for more information.
> fatal: unable to access 'https://github.com/KasRoudra/MaxPhisher/': The requested URL returned error: 403
> 
> When I try to directly run this: wget https://raw.githubusercontent.com/KasRoudra/MaxPhisher/main/maxphisher.py && python3 maxphisher.py
> I get... :
> --2024-06-05 14:57:26--  https://raw.githubusercontent.com/KasRoudra/MaxPhisher/main/maxphisher.py
> Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
> Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
> HTTP request sent, awaiting response... 404 Not Found
> 2024-06-05 14:57:27 ERROR 404: Not Found.
> 
> 
> Could someone guide me in the write direction possibly I'm missing something. I'm new and inexperienced so it could be something painfully obvious to some. Any help is much appreciated.

yo

**Comment by TJd4vhVUHBCfHDU3c1hezYgtvmj4UiqGiD** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by Rmjan10** [comment]
![Screenshot_20241028-103538](https://github.com/user-attachments/assets/3a6ab086-846c-4f7a-bc3f-fcb79810ec0a)

how can fixed this error. Please, check this screenshot?

**Comment by TJd4vhVUHBCfHDU3c1hezYgtvmj4UiqGiD** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by TJd4vhVUHBCfHDU3c1hezYgtvmj4UiqGiD** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
